<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Default Suite" time="5.221" tests="1" failures="0" errors="1">
   <testsuite name="Default Suite" tests="1" failures="0" errors="1" time="5.221" skipped="0" timestamp="2566-08-30 14:58:01" hostname="USER - DESKTOP-67R9BDO" id="Test Suites/Imported from Selenium IDE Scripts/Default Suite">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\USER\\git\\katalon\\Reports\\25660830_145753\\Imported from Selenium IDE Scripts\\Default Suite\\25660830_145753"/>
         <property name="logFiles" value="C:\\Users\\USER\\git\\katalon\\Reports\\25660830_145753\\Imported from Selenium IDE Scripts\\Default Suite\\25660830_145753\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="USER - DESKTOP-67R9BDO"/>
         <property name="os" value="Windows 11 64bit"/>
         <property name="katalonVersion" value="8.6.6.208"/>
         <property name="browser" value="Chrome 116.0.0.0"/>
         <property name="hostAddress" value="172.20.10.12"/>
         <property name="sessionId" value="2b4d69ed5febb2f04735f79029eb88c4"/>
         <property name="seleniumVersion" value="3.141.59"/>
         <property name="proxyInformation" value="ProxyInformation { proxyOption=NO_PROXY, proxyServerType=HTTP, username=, password=********, proxyServerAddress=, proxyServerPort=0, executionList=&quot;&quot;, isApplyToDesiredCapabilities=true }"/>
         <property name="platform" value="Windows 11"/>
      </properties>
      <testcase name="Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY" time="4.952" classname="Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY" status="ERROR">
         <error type="ERROR" message="Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	... 22 more
"/>
         <system-out><![CDATA[2566-08-30 14:58:02 - [TEST_CASE][ERROR] - Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY: Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	... 22 more


2566-08-30 14:58:02 - [TEST_STEP][PASSED] - "----------------------------------------------------": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - "This test case script is generated by Katalon Studio": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - "Generated date: 30-ส.ค.-2566 02:56:52 หลังเที่ยง": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - "File path: C:\Users\USER\Desktop\MYProject.side": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - "Generated by: ning.nattha6@gmail.com": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - baseUrl = "https://pos-frontend-ton-gumbjqoyzq-as.a.run.app": null

2566-08-30 14:58:02 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2566-08-30 14:58:03 - [MESSAGE][INFO] - Starting 'Chrome' driver

2566-08-30 14:58:03 - [MESSAGE][INFO] - Action delay is set to 0 milliseconds

2566-08-30 14:58:05 - [MESSAGE][PASSED] - Browser is opened with url: ''

2566-08-30 14:58:05 - [TEST_STEP][PASSED] - selenium = new com.kms.katalon.core.webui.driver.KatalonWebDriverBackedSelenium(baseUrl): null

2566-08-30 14:58:05 - [TEST_STEP][PASSED] - selenium.open("/", ""): null

2566-08-30 14:58:06 - [TEST_STEP][PASSED] - "setWindowSize | 825x818 |  | ": null

2566-08-30 14:58:06 - [TEST_STEP][ERROR] - selenium.click("css=div > .py-3 > div"): Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at Script1693382212608.run(Script1693382212608.groovy:54)
	... 13 more


2566-08-30 14:58:07 - [MESSAGE][ERROR] - Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at Script1693382212608.run(Script1693382212608.groovy:54)
	... 13 more]]></system-out>
         <system-err><![CDATA[2566-08-30 14:58:02 - [TEST_CASE][ERROR] - Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY: Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	... 22 more]]></system-err>
      </testcase>
      <system-out><![CDATA[2566-08-30 14:58:01 - [TEST_SUITE][ERROR] - Default Suite: Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	... 22 more]]></system-out>
      <system-err><![CDATA[2566-08-30 14:58:01 - [TEST_SUITE][ERROR] - Default Suite: Test Cases/Imported from Selenium IDE Scripts/Default Suite/YYTY FAILED.
Reason:
com.thoughtworks.selenium.SeleniumException: Element css=div > .py-3 > div not found
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:93)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:39)
	at com.thoughtworks.selenium.webdriven.commands.Click.handleSeleneseCommand(Click.java:26)
	at com.thoughtworks.selenium.webdriven.SeleneseCommand.apply(SeleneseCommand.java:36)
	at com.thoughtworks.selenium.webdriven.Timer.run(Timer.java:41)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.execute(WebDriverCommandProcessor.java:150)
	at com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor.doCommand(WebDriverCommandProcessor.java:76)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:198)
	at com.thoughtworks.selenium.Selenium$click$0.call(Unknown Source)
	at YYTY.run(YYTY:54)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:187)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1693382273112.run(TempTestSuite1693382273112.groovy:36)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate element even after falling back to Sizzle: div > .py-3 > div
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-67R9BDO', ip: '172.20.10.12', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_362'
Driver info: driver.version: unknown
	at com.thoughtworks.selenium.webdriven.ElementFinder.fallbackToSizzle(ElementFinder.java:154)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElementDirectlyIfNecessary(ElementFinder.java:125)
	at com.thoughtworks.selenium.webdriven.ElementFinder.findElement(ElementFinder.java:87)
	... 22 more]]></system-err>
   </testsuite>
</testsuites>
